src/content/docs/data-apis/manage-data/view-system-limits.mdx

```diff
@@ -11,9 +11,34 @@ redirects:
 
 To ensure our systems are always up and ready to support you, and to keep you from unintended use, we place [limits on the amount of data](/docs/licenses/license-information/general-usage-licenses/new-relic-data-usage-limits-policies) you can send, query, and store.
 
+## Limits UI [#limits-ui]
+
+To find the limits UI: from the [account dropdown](/docs/glossary/glossary/#account-dropdown), click **Manage your data** and click **Limits**. 
+
+This UI displays information about how close an account is to hitting rate limits (both data ingest-related limits and query limits), and violations of those limits. The data displayed here is generated from: 
+
+* [Resource use metrics](/docs/data-apis/manage-data/query-limits/#limit-metrics)
+* Limit violations reported in [`NrIntegrationError` events](/docs/data-apis/manage-data/nrintegrationerror)
+
+![Data limits UI screenshot](./images/data-limits-ui.png "Data limits UI screenshot")
+<figcaption>The limits UI displays information related to an account's data ingest rate and query rate, and violations of limits.</figcaption>
+
+What the color-coding in the violation table means: 
+* Red: exceeded a limit
+* Yellow: 80-100% of a limit
+* Green: below 80%
+* Gray: limits that don't have reported usage or violations for the given time range
+
+Some tips on using the limits UI: 
+* To get more detail about a table entry, try clicking it. Some entries have more detail, including associated NRQL queries.   
+* When you select a time range greater than 6 hrs, the chart uses average values, which may be smoothed out and not show spikes. For longer ranges, you may see the message "For the time window chosen, the 'Max limit usage' value represents the average of your usage limits." To see more accurate results, use the time picker to zoom into a time range that's less than 6 hours.
```

Comment:
This is a very clear paragraph (in that I can visualize and follow where I would go easily), but I'm wondering if you can shorten it, or if it'd be too much to add sub-bullets. It's fine as is, but maybe something to mess around with (if you want to; like I said, the content itself is good, just maybe long for a bullet point)